### PR TITLE
Increased degree limit for tangent cone computations

### PR DESCRIPTION
**Pull request by nilten**
The previous degree limit was missing some lowest order terms of obstruction equations. The change should fix this.

### CODE REVIEW

File: M2/Macaulay2/packages/VersalDeformations.m2 [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
-    Version => "2.1",
-    Date => "April 10, 2020",
+    Version => "2.2",
+    Date => "October 30, 2020",
     Authors => {
         {Name => "Nathan Ilten",
 	  HomePage => "http://www.sfu.ca/~nilten/",
 	  Email => "nilten@sfu.ca"}},
-    Keywords => {"Deformation Theory"},
+--  Keywords => {"Deformation Theory"},
```
Comment (DanGrayson) [inline M2/Macaulay2/packages/VersalDeformations.m2:15]: Perhaps you commented this out so it would run with an older version of M2?

Comment (nilten) [reply M2/Macaulay2/packages/VersalDeformations.m2:15]: Keywords is giving an error with version 1.16, which is the latest distributed version.

Comment (DanGrayson) [reply M2/Macaulay2/packages/VersalDeformations.m2:15]: Well, as this is the code to be incorporated into 1.17, let's leave the line in.

Or you could write an "if" statement based on testing the version number that would run newPackage both ways.  Then your source code would work with both versions.

**Comment by nilten** [comment]
I'm closing this pull request, and will make a new one shortly: I've made a significant number of other changes.

**Pull request closed without merging**